src/inventory_test.go

```diff
@@ -52,7 +51,7 @@ func TestGetRawInventoryEmpty(t *testing.T) {
 }
 
 func TestPopulateInventory(t *testing.T) {
-	inventory := make(sdk.Inventory)
+	inventory := inventory.New()
```

Comment:
[minor] inventory variable needs to be renamed to avoid collisions with `inventory` pkg